plugins/dnswl.js

```diff
@@ -3,8 +3,8 @@
 
 exports.register = function() {
     var plugin = this;
-    plugin.load_dnswl_ini();
     plugin.inherits('dns_list_base');
+    plugin.load_dnswl_ini();
```

Comment:
Looks like a separate issue from the queue filename. If this were in another PR and a little bit of explanation, it would likely get merged straight away.